src/cli/cbenv_register.rs

```diff
@@ -230,6 +230,7 @@ pub fn update_config_file(guard: &mut MutexGuard<State>, span: Span) -> Result<(
             c.access_key(),
             Some(c.timeout()),
             c.default_project(),
+            None,
```

Comment:
Why is this None?
Presumably we would want to save the api_endpoint to the config file?

Reply:
I don't think that we do. Since either the endpoint will be the default, so adding the default endpoint to the file will just cause confusion. Or it will not be the default, in which case it will already be in the config file.

Reply:
Ah, I see. Since it's only configurable through the config file, it doesn't make sense to update the config file.